user-manual.md

```diff
@@ -0,0 +1,47 @@
+# A high level description of RecipeCart:
+This system is a website for uploading, searching, and rating recipes. Using our descriptive recipe tagging system (missing), users can specify dietary restrictions or preferences that match the recipes they wish to search for. Users can also leave ratings (missing) on recipes to let others know the average quality of the recipe.  
+
+# How to install the software:
+To install the software, there are separate requirements for the frontend and the backend. After cloning the repo to your local system, follow the steps below:
+- Frontend:
+    - Install Node.js (v18) from the Node website
+    - In the `/front-end/` directory, run `npm i` to install dependencies
+- Backend:
+    - 
+
+# How to run the software:
+After installing all the necessary dependencies, follow the steps below to run the software:
+- Frontend:
+    - In the `/front-end/` directory, run `npm run build`
+    - Run `npm run start` to launch the website
+    - Access the website at `http://localhost:8080`
+- Backend:
+    - 
+
+# How to use the software:
+Once the dependencies are installed and the website/backend are running, here is the functionality you can utilize:
+- Recipe Search
+    - Search the database for recipes using the search bar
```

Comment:
Since you are addressing the user, who might not necessarily understand what a database is, a better way to phrase this might be "search for recipes by typing in ingredients in the search bar at the top of the home page"